bin/build

```diff
@@ -6,7 +6,7 @@ set -ex
 
 . bin/build_utils
 
-CURRENT_DIR=$($(dirname $0)/abspath)
+CURRENT_DIR=$("$(dirname "$0")/abspath")
```

Comment:
thoughts about setting `prefix=$(dirname "$0")` before this line? quotes inside quotes look weird to me